### ISSUE TITLE
-Wimplicit-function-declaration in drivers/net/ipa/gsi.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#963]
```
# 00:10:04 drivers/net/ipa/gsi.c:220:39: error: implicit declaration of function 'field_max' [-Werror,-Wimplicit-function-declaration]
```
Linaro TCWG is reporting that mainline(!!!) regressed due to the above.

Looks like there was a patch sent yesterday.
https://lore.kernel.org/netdev/20200311024240.26834-1-elder@linaro.org/

and it was reported:
1. https://lore.kernel.org/lkml/cdadf72c-0233-91f4-73a5-cee9636d32cc@linaro.org/
2. https://lkml.org/lkml/2020/3/10/1839

**Comment by nickdesaulniers** [comment]
https://groups.google.com/g/clang-built-linux/c/pX-kr_t5l_A

**Comment by nickdesaulniers** [comment]
reported: https://lore.kernel.org/netdev/20200401173515.142249-1-ndesaulniers@google.com/

**Comment by nickdesaulniers** [comment]
fix should be in mainline, so IDKWTF: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=e31a50162feb352147d3fc87b9e036703c8f2636

**Comment by nathanchance** [comment]
This is another one of those instances where Linaro’s CI continues to report regressions that have long since been fixed... I really don’t know why happens. If I had to guess, they might be building every commit in isolation because the first good build is 530f9216a953, which is a rather random commit to test. I don’t know though... it makes it really annoying to reliably triage issues because then we end up doing what we did today and make someone think there was a regression when there really wasn’t.
